recipe/meta.yaml

```diff
@@ -29,11 +29,11 @@ outputs:
         - cartopy >=0.21
         - cf-units >=3.1
         - cftime >=1.5
-        - dask-core >=2022.9.0
+        - dask-core >=2022.9.0, !=2024.8.0
         - libnetcdf !=4.9.1
-        - matplotlib-base >=3.5
+        - matplotlib-base >=3.5, !=3.9.1
         - netcdf4
-        - numpy >=1.24,!=1.24.3,<2
+        - numpy >=1.24,!=1.24.3
```

Comment:
very happy to see this! :beer: @bouweandela - we're in business with our PR :+1: